### PR TITLE
Removing Bay plane usage.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Basically Bay was using planes as super-layers, which is clunky and limiting since it stops us using things like plane masters. This frees things up for future ports of Lohi's multiz and client-based plane master usage.

### CODE REVIEW

File: code/_helpers/icons.dm [hunk 1]
```diff
@@ -107,7 +107,7 @@ AngleToHue(hue)
 	Converts an angle to a hue in the valid range.
 RotateHue(hsv, angle)
 	Takes an HSV or HSVA value and rotates the hue forward through red, green, and blue by an angle from 0 to 360.
-	(Rotating red by 60� produces yellow.) The result is another HSV or HSVA color with the same saturation and value
+	(Rotating red by 60� produces yellow.) The result is another HSV or HSVA color with the same saturation and value
```
[the remark this answers is not in the document]
Comment (MistakeNot4892) [reply code/_helpers/icons.dm]: Not sure why VSC has decided to do that.

[the remark this answers is not in the document]
Comment (EmperorJon) [reply code/_helpers/icons.dm]: Well that sure is one way to deal with it

File: code/game/turfs/space/space.dm [hunk 2]
```diff
@@ -13,12 +12,11 @@
 	LAZYINITLIST(dust_cache)
 	for (var/i in 0 to 25)
 		var/image/im = image('icons/turf/space_dust.dmi',"[i]")
-		im.plane = DUST_PLANE
+		im.layer = DUST_LAYER
```
Comment (Lohikar) [inline code/game/turfs/space/space.dm]: I have a feeling that this and the SPACE_PLANE change may break parts of skybox and/or its parallax.

Comment (MistakeNot4892) [reply code/game/turfs/space/space.dm]: Currently dust isn't showing up, I'm quite confused as to why.

Comment (MistakeNot4892) [reply code/game/turfs/space/space.dm]: This is fixed.

**Pull request merged**